src/screens/ParadeMapScreen/Map.js

```diff
@@ -119,10 +119,8 @@ class Map extends PureComponent<Props, State> {
         this.setState({
           moveToUserLocation: false
         });
-        Alert.alert(
-          "We couldn't find your location",
-          "GPS or other location finding magic might not be available, please try again later"
-        );
+
+        this.messageBannerRef.current.handleAnimation();
```

Comment:
My only suggestion here would be to rename handleAnimation to something more appropriate for external consumers of the MessageBanner class. For example `show()`.